.github/workflows/refund-nightly-skyline.yml

```
@@ -0,0 +1,39 @@
+---
+name: Refund Nightly Local Tests
```

```suggestion
name: Refund Nightly Skyline Local Tests
```
change name of the workflow